.github/workflows/cicd.yml

```diff
@@ -70,7 +70,7 @@ jobs:
           path: storybook-static
 
       - name: Publish to Chromatic
-        uses: chromaui/action@v1
+        uses: chromaui/action@v10
```

Comment:
Ik zie in deze PR geen publish naar Chromatic + UI tests, kan dat hier door komen?

Reply:
Nee, ik had alleen de status van de PR van draft naar ready for review veranderd maar er was nog een extra (force) push nodig.

Reply:
Aah oké top, dan merge ik m! Bedankt!